app/database/migrations/20190225134140_users.js

```diff
@@ -21,6 +21,8 @@ exports.up = function (knex, Promise) {
     tbl.string('instagram', 150);
     tbl.string('phone_number', 50);
     tbl.string('roles', 50).notNullable();
+    tbl.integer('total_stars');
+    tbl.integer('total_reviews');
```

Comment:
**issue**: We can't edit existing migrations, they're designed to only run once. Create a new migration with `npx knex migrate:make` and add these fields in there.